### DISCUSSION TITLE
UCB CS61A: Structure and Interpretation of Computer Programs - CSDIY.wiki

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#498] [Announcements]
# UCB CS61A: Structure and Interpretation of Computer Programs - CSDIY.wiki

CS自学指南

https://csdiy.wiki/en/%E7%BC%96%E7%A8%8B%E5%85%A5%E9%97%A8/CS61A/

**Comment by zcsrssrs** [comment]
google cs61a，你的网站竟然是第三个结果，排在官网和uc berkeley之后！！

**Reply by DanielDaniel2201** [reply]
质量太高了😂

**Comment by hhz0328** [comment]
2025打个卡

**Comment by safanasjev** [comment]
List of CS61A Study Guides:
https://cs61a.org/study-guide/functions-control/
https://cs61a.org/study-guide/environments-hof/
https://cs61a.org/study-guide/self-reference/
https://cs61a.org/study-guide/recursion/
https://cs61a.org/study-guide/trees/
https://cs61a.org/study-guide/mutation/
https://cs61a.org/study-guide/iterators/
https://cs61a.org/study-guide/objects/
https://cs61a.org/study-guide/mutable-trees/
https://cs61a.org/study-guide/orders-of-growth/
https://cs61a.org/study-guide/scheme-fp/
https://cs61a.org/study-guide/interpreters/
https://cs61a.org/study-guide/macros/
https://cs61a.org/study-guide/sql/
https://cs61a.org/study-guide/regex/
https://cs61a.org/study-guide/bnf/
